app/services/proofing/mock/resolution_mock_client.rb

```diff
@@ -3,9 +3,17 @@ module Mock
     class ResolutionMockClient < Proofing::Base
       vendor_name 'ResolutionMock'
 
-      required_attributes :first_name, :ssn, :zipcode
-
-      optional_attributes :uuid, :uuid_prefix
+      required_attributes :uuid,
+                          :first_name,
+                          :last_name,
+                          :dob,
+                          :ssn,
+                          :address1,
+                          :city,
+                          :state,
+                          :zipcode
+
+      optional_attributes :address2, :uuid_prefix, :dob_year_only
```

Comment:
Syncing this to the real vendor implementation should help surface the issue in feature specs.

https://github.com/18F/identity-idp/blob/fee0abe793255df5ce2b0eb255508c758e75451b/app/services/proofing/lexis_nexis/instant_verify/proofer.rb#L8-L18